src/shared/scrapers/IN/index.js

```diff
@@ -77,8 +77,8 @@ const scraper = {
       const data = {
         state: stateMapped,
         cases: parse.number($tr.find('td:nth-child(3)').text()),
-        deaths: parse.number($tr.find('td:nth-child(6)').text()),
-        recovered: parse.number($tr.find('td:nth-child(5)').text())
+        recovered: parse.number($tr.find('td:nth-child(4)').text()),
+        deaths: parse.number($tr.find('td:nth-child(5)').text())
```

Comment:
I believe that these column re-orderings may _not_ be backwards compat with cached data ... if the columns were in a different order in the past.  See the docs at https://github.com/covidatlas/coronadatascraper/blob/master/docs/sources.md#what-to-do-if-a-scraper-breaks .  If I'm wrong, LMK!  Thank you very much for the contribution! jz